packages/app/src/runs/RunsSkeletonRow.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <div
+    data-cy="runsSkeletonRow"
+    class="p-[16px] gap-[16px] flex justify-between children:flex"
+  >
+    <div class="shrink-1 overflow-hidden gap-[8px] children:rounded-lg children:h-[16px] children:bg-gray-50">
```

Comment:
>
```suggestion
    <div class="overflow-hidden gap-[8px] children:rounded-lg children:h-[26px] children:bg-gray-50">
```
Height of 26px will match actual code